src/osgEarthDrivers/gdal/ReaderWriterGDAL.cpp

```diff
@@ -664,6 +674,22 @@ build_vrt(std::vector<std::string> &files, ResolutionStrategy resolutionStrategy
                 GDALSetRasterNoDataValue(hBand, NO_DATA_VALUE);
         }
 
+        // Join the products coordinates
+        std::ostringstream oss;
+        for (const auto &tileExtent : tilesExtent) {
+            oss << tileExtent << ';';
+        }
+
+        // Add the products coordinates to the metadata
+        const std::string tilesExtentString = oss.str();
+        char **tilesExtentStringList = nullptr;
+        tilesExtentStringList = CSLSetNameValue(tilesExtentStringList, "TILES_EXTENT", tilesExtentString.c_str());
+        GDALRasterBandH hBand;
+        GDALAddBand(hVRTDS, GDT_Unknown, NULL);
```

Comment:
We shall not create a band in the VRT!
That may lead to many errors and that 's not clean.

You may implement this specific case in getBandAllMetaData()
Or the better solution is to add tilesExtent property in the layerService so we dont fake the structures...

We can have a call if you want